mango-cli/src/main/scala/org/bdgenomics/mango/cli/VizReads.scala

```diff
@@ -718,6 +724,30 @@ class VizReads(protected val args: VizReadsArgs) extends BDGSparkCommand[VizRead
     }
 
     /**
+     * Runs total data scan over all feature and variant files satisfying a certain predicate.
+     *
+     * @return Returns list of regions in the genome satisfying predicates
+     */
+    def discoverGnocchi(): List[ReferenceRegion] = {
+
+      // filtering for features
+      val featureRegions: Option[RDD[(ReferenceRegion, Long)]] = {
+        if (!VizReads.featuresExist) {
+          log.warn("specified discovery predicate for features but no variant files were provided")
+          None
+        } else {
+          var features: RDD[Feature] = sc.parallelize[(Feature)](Array[(Feature)]())
+          VizReads.featureData.get.files.foreach(fp => features = features.union(FeatureMaterialization.load(sc, None, fp).rdd))
```

Comment:
Ok we should fix this then. If we are going to differentiate discoverGnocci() from discoverFeatures() the methods should be distinct